readme.md

```diff
@@ -274,6 +274,8 @@ yarn type-check
 ## Lighthouse Integrations
 This section details services that have integrated Lighthouse data. If you're working on a cool project integrating Lighthouse and would like to be featured here, file an issue to this repo or tweet at us [@_____lighthouse](https://twitter.com/____lighthouse?lang=en)!
 
+* **[Lighthouse Keeper](https://lighthouse-keeper.com/)** - Lighthouse Keeper monitors your pages' Lighthouse scores and notifies you if they drop below your thresholds. You can monitor 3 URLs once a day, for free!
```

Comment:
this is an alphabetized list, mind putting yours after HTTPArchive?

Reply:
also, the others tend to stay away from exclamatory marketing statements ;) 

maybe just `Lighthouse Keeper is a free service that monitors up to 3 URLs once per day.`?

Reply:
Sorry, didn't notice it was alphabetized! I also changed the description as requested. Please resolve and merge if you are happy with this.

Cheers